Dockerfile

```diff
@@ -22,8 +22,7 @@ RUN yum -y groupinstall "Development Tools"
 
 # Python 3.6 update:
 RUN yum update -y \
-    && yum install -y https://centos7.iuscommunity.org/ius-release.rpm \
-    && yum install -y python36u python36u-libs python36u-devel python36u-pip \
+    && yum install -y python36 python36-libs python36-devel python36-pip \
```

Comment:
If not relying on side effects of order, it'd be good to consolidate the below `yum install` lines into a single command invocation.